Travel_fellowships.md

```diff
@@ -144,13 +141,62 @@ This will not be a limiting factor if applicants are new to bioinformatics and o
+
+We ask all reviewers to finish the review with a few summary sentences to describe the application and the motivations for their scores to facilitate conversation during the selection panel discussion.
+Below we provide a copy of our review form.
+
+### Event and nature of applicant's participation
+
+- [ ] 1 (criteria not met) Does not provide enough information or seems to misunderstand what kind of event OBF Event fellowship supports
+- [ ] 2 (criteria partially met) The event or the nature of their participation may not directly relate to Open Source in bioinformatics, but they have some understanding/plan on how they would use the knowledge gained at this event to promote open source practices and diversity
+- [ ] 3 (criteria fully met) Seems to have a clear understanding of how their participation in this event relates to open-source bioinformatics or open science in general, and how they will further use and promote the knowledge obtained at this event
+
+### Open Source Resources
+- [ ] 1 (criteria not met) Shares none or unrelated/irrelevant information on open source resource and how they relate to their proposed event
+- [ ] 2 (criteria partially met) Shares some relevant information on open source resource and how they relate to their proposed event
+- [ ] 3 (criteria fully met) Shares appropriate and relevant information on open source resource and how they relate to their proposed event
+
+### Promoting diversity and representation of historically excluded or underrepresented groups *
+- [ ] 1 (criteria not met) The application lacks information or any justification on how their participation will help promote diversity and representation
+- [ ] 2 (criteria partially met) The application provides little information or some justification on how their participation will help promote diversity and representation
+- [ ] 3 (criteria fully met) The application provides sufficient information or justification on how their participation will help promote diversity and representation
+
+### Justification for funding
+- [ ] 1 (criteria not met) The requested funding does not fit our criteria of qualified items that can be funded through this fellowship and the applicant has not provided enough justification
+- [ ] 2 (criteria partially met) The requested funding partially fits our criteria of qualified items that can be funded through this fellowship and the applicant has provided enough justification
+- [ ] 3 (criteria fully met) The requested funding completely fits our criteria of qualified items that can be funded through this fellowship and the applicant has provided sufficient justification
+
+### Would you recommend this applicant for OBF Event Fellowship? *
+- [ ] No
+- [ ] Yes
+- [ ] Unsure/Request to discuss
```

Comment:
Why the * here?

Reply:
i think that may be a relic caused by copying the questions from the review form. required (non-optional) questions are marked there with *.

Reply:
Ah. Maybe added [required] to the text, or just drop the asterisk?